pkg/runctl/config.go

```diff
@@ -43,10 +47,21 @@ func loadPipelineRunsConfig(clientFactory k8s.ClientFactory) (*pipelineRunsConfi
 		return nil, err
 	}
 
+	networkMap, err := configMapIfce.Get(networkPoliciesConfigMapName, metav1.GetOptions{})
+	// Do we need this only for testing?
+	if k8serrors.IsNotFound(err) {
+		return &pipelineRunsConfigStruct{}, nil
+	}
+	if err != nil {
+		return nil, err
+	}
+
+	defaultNetworkPolicyKey := networkMap.Data[networkPoliciesConfigKeyDefault]
 	config := &pipelineRunsConfigStruct{
-		NetworkPolicy: configMap.Data[pipelineRunsConfigKeyNetworkPolicy],
-		LimitRange:    configMap.Data[pipelineRunsConfigKeyLimitRange],
-		ResourceQuota: configMap.Data[pipelineRunsConfigKeyResourceQuota],
+		LimitRange:           configMap.Data[pipelineRunsConfigKeyLimitRange],
+		ResourceQuota:        configMap.Data[pipelineRunsConfigKeyResourceQuota],
+		NetworkPolicies:      networkMap.Data,
+		DefaultNetworkPolicy: networkMap.Data[defaultNetworkPolicyKey],
```

Comment:
`NetworkPolicies` should not contain the special keys (those starting with underscore). We should reserve any name beginning with underscore for future special meanings, and ignore them here.

Isn't it a config error if there's more than one network policy defined, but no default? Then we should fail here.